Classes/GHNSURL+Utils.m

```diff
@@ -62,86 +53,23 @@ + (NSArray *)gh_dictionaryToQueryArray:(NSDictionary *)queryDictionary sort:(BOO
     }
     
     if (!valueDescription) continue;
-    
-    if (encoded) key = [self gh_encodeComponent:key];
-    if (encoded) valueDescription = [self gh_encodeComponent:valueDescription];
-    [queryStrings addObject:[NSString stringWithFormat:@"%@=%@", key, valueDescription]];
+    NSURLQueryItem *item = [NSURLQueryItem queryItemWithName:key value:valueDescription];
```

Comment:
Like you've pointed out, there's a behavior change here ("foo+bar" no longer becomes "foo%2Bbar"). Do we need to preserver that old behavior? If not, I feel a little uncomfortable releasing this as the same method but without those old quirks, especially when its utility is questionable in a world with `NSURLComponents`.

If we don't need to preserve the old behavior, should we just delete `gh_dictionaryToQueryString:` and `gh_queryString`?